confidant_client/__init__.py

```diff
@@ -404,6 +404,41 @@ def get_credential(self, id):
         ret['result'] = True
         return ret
 
+    def get_credential_services(self, id):
+        """Get a credential's services from ID."""
+        # Return a dict, always with an attribute that specifies whether or not
+        # the function was able to successfully get a result.
+        ret = {'result': False}
+
+        # Make a request to confidant with the provided url, to fetch the
```

Comment:
let's replace this comment with what we're actually getting here, which is the list of services using the cred